apps/web/src/app/next_api/kanttiinit/route.ts

```diff
@@ -0,0 +1,41 @@
+import type {
+  OpeningHour,
+  Restaurant,
+} from "../../../lib/types/kanttiinit-types";
+
+interface RestaurantResponse {
+  openingHours: OpeningHour[];
+  id: number;
+  type: string;
+  url: string;
+  latitude: number;
+  longitude: number;
+  address: string;
+  priceCategory: string;
+  name: string;
+}
+
+export async function GET() {
+  const response: Response = await fetch(
+    "https://kitchen.kanttiinit.fi/restaurants?lang=fi&ids=2,7,52&priceCategories=student",
+  );
+
+  if (response.status !== 200) {
+    return { status: response.status, result: null };
```

Comment:
If we replace this line with `return Response.error();` the typecheck error goes away